src/foundry_dev_tools/__init__.py

```diff
@@ -1,14 +1,20 @@
 """Foundry Clients developed at Merck KGaA, Darmstadt, Germany."""
-from .__about__ import __version__
-from .cached_foundry_client import CachedFoundryClient
-from .config import FOUNDRY_DEV_TOOLS_DIRECTORY, INITIAL_CONFIG, Configuration
-from .foundry_api_client import FoundryRestClient
+
+from foundry_dev_tools.__about__ import __version__
```

Comment:
It seems the `__about__` file is only created after `pip install -e .` which makes running the tests from PyCharm impossible without installing the package first. That was better previously. Can you find a better solution that allows running tests without installing into the local environment first? Or convince me that this is now best practice in python packaging ;)